modules/govcms_ckan_display/plugins/visualisation/spline_chart.inc

```diff
@@ -133,6 +136,22 @@ function govcms_ckan_display_spline_chart_configure($plugin, $form, $form_state,
     '#default_value' => govcms_ckan_get_config_value($config, 'area'),
   );
 
+  $area_opacity_options = array(10, 20, 30, 40, 50, 60, 70, 80, 90, 100);
+  $config_form['area_opacity'] = array(
+    '#type' => 'select',
+    '#title' => t('Area Opacity'),
+    '#default_value' => govcms_ckan_get_config_value($config, 'area_opacity'),
+    '#options' => array_combine($area_opacity_options, $area_opacity_options),
+    '#states' => array(
+      'visible' => array(
+        // This field is deeply nested so resorting to a fairly ugly selector
+        // if selector is not found the in the edge case that the lang or delta
+        // is different field will fallback to being visible anyway.
+        ':input#edit-ckan-visualisation-und-0-config-visualisation-config-area' => array('checked' => TRUE),
```

Comment:
I know :( I could have just omitted this but admin form is getting really overwhelming so any way to simplify i think is worth the tradeoff.